modules/setting-up-topology-manager.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * scaling_and_performance/using-topology-manager.adoc
+
+[id="seting_up_topology_manager_{context}"]
+= Setting up Topology Manager
+
+.Prequisites
+* Configure the CPU Manager policy to be `static`. Refer to Using CPU Manager in the Scalability and Performance section.
+
+.Procedure
+
+. Enable the LatencySensitive FeatureGate
++
+----
+# oc edit featuregate/cluster
+----
+
+. Add Feathre Set: LatencySensitive to the spec:
+// todo, identify the features LatencySensitive enable
```

Comment:
@vpickard is there anything you want to add now for this todo, or is this for 4.4 or later?

Reply:
@lamek This was just more of a note to myself that it would be nice to identify the individual features/config that will be set when the LatencySensitive feature set is enabled. Let me see if I can figure that out now.

BTW, looks like I had a typo on line 19 above.

Reply:
OK, for reference, the LatencySensitive feature set is defined here:
https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L127


After reading this again, I think it will be sufficient to just remove the todo.